docs/index.rst

```diff
@@ -50,8 +50,7 @@ Contents:
 
    prod-requirements
    prod-install
-   prod-auth-first-login
-   prod-health-check-debug
+   prod-troubleshooting
    prod-customize
    prod-maintain-secure-upgrade
    prod-authentication-methods
```

Comment:
The new page `ssl-certificates` isn't currently part of the ToC. Might want to add it, depending on how visible/discoverable it should be.